src/main/java/automation/guru/hooks/Hooks.java

```diff
@@ -91,6 +91,7 @@ public void setup(Scenario scenario) throws MalformedURLException {
                 webDriver = new RemoteWebDriver(new URL(URL), caps);
             }
         }
+        // Test commit
     }
```

Comment:
isToken is declared as var, can we use type as let instead? Also I see that method signOnwihToken() is returning null object in line 128, once this return is encountered, the rest of the code for setting cookie is never executed. We need a better way of handling.

Reply:
Fixes done , Please review again.

Reply:
Okay,